packages/drivers/odsp-driver-definitions/src/resolvedUrl.ts

```diff
@@ -15,6 +15,23 @@ export interface IOdspUrlParts {
 	itemId: string;
 }
 
+/**
+ * Information about the file to be migrated.
```

Comment:
We don't need to specify migration specifically here as this is just file metadata. You can specify the migration when you will be setting the If-Match header.